avalon/blender/ops.py

```diff
@@ -0,0 +1,392 @@
+"""Blender operators and menus for use with Avalon."""
+
+import os
+import sys
+from distutils.util import strtobool
+from pathlib import Path
+from pprint import pformat
+from types import ModuleType
+from typing import Dict, List, Optional, Union
+
+import bpy
+import bpy.utils.previews
+
+import avalon.api as api
+from avalon.vendor.Qt import QtCore, QtWidgets
+from avalon.vendor.Qt.QtWidgets import (
+    QApplication,
+    QDialog,
+    QHBoxLayout,
+    QLabel,
+    QPushButton,
+    QVBoxLayout,
+)
```

Comment:
I believe Avalon code style recommends not to import these widgets like this. Probably @mottosso can be best confirm at this stage.

Reply:
Yes, please use `from avalon.vendor.Qt import QtWidgets`. Namespaces are good.